src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        String allTextFile = readFile(fromFileName);
+        supply = calculetedSupplyForReport(allTextFile);
+        buy = calculetedBuyForReport(allTextFile);
+        writeToFile(supply,buy,toFileName);
+    }
```

Comment:
With current approach you have to go through file content twice what is not optimal.
Please replace calculetedSupplyForReport and calculetedBuyForReport with single method that will take all information with one loop though file content.
Also it would be better to move there logic that forms report from writeToFile method.
Overall from public method it should be something like below:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String fileContent = readFile(fromFileName);
        String report = generateReport(fileContent);
        writeToFile(report, toFileName);
    }
```